README.md

```diff
@@ -9,28 +9,30 @@ HTTP RESTFul for calling DataWow APIs
 gem 'datawow', '~> 1.3.2'
 ```
 
-#### Generate setting
+### Rails
 
 ```console
 $ rails generate datawow:install
 ```
-The generator will install an initializer which describes ALL of library's configuration options. It is *imperative* that you take a look at it. When you are done, you are ready to call it to any of your codes.
-
-**Note**: Our library work with Rails 4.1 or above
+**Note**: Rails 4.1 or above
 
 # Usage
 
-To call our module use `Datawow` and follow by class of APIs which we're going to explain what API we have
+To call our module use `Datawow` then follow by class of APIs which we're going to explain what API we have
```

Comment:
To call our module, use `Datawow` followed by any of the classes provided in our APIs